src/scss/includes/panels/categories.scss

```diff
@@ -6,45 +6,40 @@
 
 .category__panel__pj {
   width: 100%;
-  height: 50px;
-  padding: 0 20px;
-  display: flex;
-  line-height: 18px;
-  align-items: center;
-  justify-content: space-between;
-
-  &_title {
-    font-weight: normal;
-    color: $primary_text;
-    background: url('../images/pagesjaunes.svg') no-repeat left center;
-    background-size: 18px;
-    padding-left: 24px;
-  }
-
-  &_partnership {
-    display: inline-flex;
-    align-items: center;
-    font-size: 12px;
-    color: $secondary_text;
-
-    .icon-icon_info {
-      font-size: 16px;
-      margin-right: 4px;
-    }
-  }
+  height: 36px;
+  position: absolute;
+  bottom: -36px;
+  border-top: 1px solid $grey-light;
+  background: $background;
+  line-height: 36px;
+  text-align: center;
+  font-size: 14px;
+  color: $grey-semi-darkness;
+  border-radius: 0 0 12px 12px;
 }
 
 .category__panel__items {
   animation: appear 600ms forwards;
 }
 
 .category__panel__error {
-  padding: 16px 16px 32px 16px
+  padding: 16px 16px 32px 16px;
+
+  & ~ .category__panel__pj {
+    display: none;
+  }
 }
 
 @media (max-width: 640px) {
   .category__panel__pj {
-    height: 20px;
+    position: fixed;
```

Comment:
It should be part of the panel content.
By moving it below instead of inside, you're making your life very complicated (having to change complex panel height computations and such, introducing a lot of new rules, etc.) and new bugs are appearing:

No border radius:

![Capture d’écran de 2020-09-24 11-26-32](https://user-images.githubusercontent.com/243653/94127604-6b105b80-fe59-11ea-9c2d-db168abffbd5.png)


No shadow:

![Capture d’écran de 2020-09-24 11-26-19](https://user-images.githubusercontent.com/243653/94127699-83807600-fe59-11ea-90de-1c702b3abd47.png)